plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/query/PinotSqlFormatter.java

```diff
@@ -155,6 +156,14 @@ private static String formatFilter(FilterContext filterContext, Context context)
         throw new PinotException(PINOT_INVALID_PQL_GENERATED, Optional.empty(), format("Unexpected filter type: '%s'", filterContext.getType()));
     }
 
+    static String formatOptions(Map<String, String> queryOptions)
+    {
+        String options = queryOptions.keySet().stream()
+                .map(key -> " OPTION(" + key + "=" + queryOptions.get(key) + ")")
+                .collect(Collectors.joining(","));
```

Comment:
nit: use a space after the comma
```
Collectors.joining(", ")
```

Reply:
By doing this we I'm ending with double space between options as I already add one in from of the keyword OPTION line 162